lesson05/assignment/index.html

```diff
@@ -9,7 +9,56 @@
     </head>
     <body>
         <main class="main">
-          
+            <div class="grid ">
```

Comment:
While this setup works, we see it's a whole sort of `<div>`s. We've got some more semantic elements we can use to describe out content, take a look at swapping out your header/footer/sidebar divs with header/footer/aside elements.